packages/utils/details-page/DetailsPage.tsx

```diff
@@ -82,7 +82,7 @@ const DetailsPage: React.FC<DetailsPageProps> = ({
           }
           actions={actions}
           resource={resource}
-          className="odf-resource-details"
+          className="resource-details"
```

Comment:
same here, the class name needs a prefix. You can use mcgms

Reply:
changed to mcgms-resource-details